frontend/src/scenes/products/Products.tsx

```diff
@@ -21,6 +26,57 @@ export function getProductIcon(color: string, iconKey?: string | null, className
     return <Icon className={className} color={color} />
 }
 
+export function SelectableProductCard({
+    product,
+    productKey,
+    onClick,
+    orientation = 'vertical',
+    className,
+    selected = false,
+}: {
+    product: OnboardingProduct
+    productKey: string
+    onClick: () => void
+    orientation?: 'horizontal' | 'vertical'
+    className?: string
+    selected?: boolean
+}): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const onboardingCompleted = currentTeam?.has_completed_onboarding_for?.[productKey]
+    const vertical = orientation === 'vertical'
+    return (
+        <LemonCard
+            data-attr={`${productKey}-onboarding-card`}
+            className={clsx('flex justify-center cursor-pointer', vertical ? 'flex-col' : 'items-center', className)}
+            key={productKey}
+            onClick={onClick}
+            focused={selected}
+        >
+            {onboardingCompleted && (
+                <Tooltip
+                    title="You've already set up this product. Click to return to this product's page."
+                    placement="right"
+                >
+                    <div
+                        className="relative"
+                        onClick={(e) => {
+                            e.stopPropagation()
+                            router.actions.push(getProductUri(productKey as ProductKey))
+                        }}
+                        data-attr={`return-to-${productKey}`}
+                    >
+                        <Icons.IconCheckCircle className="absolute top-0 right-0" color="green" />
```

Comment:
```suggestion
                        <IconCheckCircle className="absolute top-0 right-0" color="green" />
```

better to use this directly, you'll need to update the import. Though I realize this wasn't your code originally.

Reply:
ace9fa7716